astropy/io/votable/tests/converter_test.py

```diff
@@ -65,6 +66,56 @@ def test_unicode_mask():
     assert c.output("Foo", True) == ''
 
 
+def test_unicode_as_char():
+    config = {'verify': 'exception'}
+    field = tree.Field(
+        None, name='unicode_in_char', datatype='char',
+        arraysize='*', config=config)
+    c = converters.get_converter(field, config=config)
+
+    # Test parsing.
+    c.parse('XYZ')  # ASCII succeeds
+    with pytest.raises(
+            exceptions.W55,
+            match=r'FIELD \(unicode_in_char\) has datatype="char" but contains non-ASCII value'):
+        c.parse("zła")  # non-ASCII
+
+    # Test output.
+    c.output('XYZ', False)  # ASCII str succeeds
+    c.output(b'XYZ', False)  # ASCII bytes succeeds
+    value = 'zła'
+    value_bytes = value.encode('utf-8')
+    with pytest.raises(
```

Comment:
I wonder if these two `pytest.raises` should be in a loop, but given `n=2`, maybe not? I'll leave this for you to decide.

Reply:
Good question.  Since the second value to test is derived from the first, I think I'll leave this as is for now.